images/build/debian-iptables/buster/Dockerfile

```diff
@@ -14,10 +14,11 @@
 
 FROM BASEIMAGE
 
-# Install latest iptables package from buster-backports
+# Install specified iptables package from buster-backports
+ARG IPTABLES_VERSION
 RUN echo deb http://deb.debian.org/debian buster-backports main >> /etc/apt/sources.list; \
     apt-get update; \
-    apt-get -t buster-backports -y --no-install-recommends install iptables
+    apt-get -t buster-backports -y --no-install-recommends install iptables=${IPTABLES_VERSION}*
```

Comment:
Do we really want to version lock like this?  I'm honestly asking - previously we just took whatever was latewst